datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDatePickerViewHolderFactory.kt

```diff
@@ -218,20 +273,55 @@ internal object QuestionnaireItemDatePickerViewHolderFactory :
   }
 }
 
-internal const val TAG = "date-picker"
-internal val ZONE_ID_UTC = ZoneId.of("UTC")
+internal fun formatDate(localDate: LocalDate?, acceptableDateFormat: String?): String {
+  localDate?.let {
+    if (!acceptableDateFormat.isNullOrEmpty()) {
+      val formatter: DateTimeFormatter = DateTimeFormatter.ofPattern(acceptableDateFormat)
+      return formatter.format(localDate)
+    } else {
+      return it.localizedString
+    }
+  }
+  return ""
+}
```

Comment:
is it okay to have return type String? if yes, can you please try below suggestion ?

```suggestion
  internal fun formatDate(localDate: LocalDate?, acceptableDateFormat: String?) =
    localDate?.let {
      if (acceptableDateFormat.isNullOrEmpty()) {
        it.localizedString
      } else {
        DateTimeFormatter.ofPattern(acceptableDateFormat).format(localDate)
      }
    }
```
Or if you need return type String only, you can try below suggestion

```
internal fun formatDate(localDate: LocalDate?, acceptableDateFormat: String?): String {
    localDate?.let {
      return if (acceptableDateFormat.isNullOrEmpty()) {
        it.localizedString
      } else {
        DateTimeFormatter.ofPattern(acceptableDateFormat).format(localDate)
      }
    }
    return ""
  }
```